src/GenesisWindow.cpp

```diff
@@ -93,9 +93,12 @@ GenesisWindow::GenesisWindow() :
 	menu->AddItem(new BMenuItem("Make dir...",	new BMessage(MENU_COMMANDS_MKDIR), 0));
 	menu->AddItem(new BMenuItem("Delete...",	new BMessage(MENU_COMMANDS_DELETE), 0));
 	menu->AddSeparatorItem();
+	menu->AddItem(new BMenuItem("Open dir in target panel" , new BMessage(MENU_COMMANDS_DIR_TARGET), '.'));
+	menu->AddSeparatorItem();
 	menu->AddItem(new BMenuItem("Create link on Desktop..." , new BMessage(MENU_COMMANDS_CREATE_SYMLINK), 'L'));
 	menu->AddSeparatorItem();
-	menu->AddItem(new BMenuItem("Open terminal window", new BMessage(MENU_TERMINAL), 'T', B_OPTION_KEY));
+	menu->AddItem(new BMenuItem("Open Tracker window", new BMessage(MENU_TRACKER), 0));
```

Comment:
Suggestion: Put the two new commands above the "Create link on Desktop" and rename to
* "Open dir in Tracker"
* "Open dir in Terminal"
That would look nicely "symmetrical"

Reply:
@humdingerb I have moved the command above "Create link" - thx, it looks better now.
But I have not renamed them because the "dir" would then refferer to 2 different stuff. Because opening terminal/tracker always opens current path of the panel, whereas "open dir in target panel" opens selected directory (or current directory if dir is not selected)

Reply:
Would it be OK to have identical behaviour for the Tracker/Terminal commands? i.e. open the currently selected dir  in Tracker/Terminal, or if the selected item isn't a dir, open the current panel dir in Tracker/Terminal.

If that's not OK, how about "Open current panel in Tracker/Terminal"?

Reply:
For me it is more usefull to have option to open current panel in terminal/tracker. I will update the labels.

Reply:
Fair enough. Let's merge! :)